requirements/local.txt.sample

```diff
@@ -0,0 +1,16 @@
+# This file is meant to be customized for personal development environments.
+# Feel free make a copy without the .sample extension to use locally.
+#
+# requirements/local.txt is in .gitignore so you will not be prompted to (nor
+# should you) commit your custom version of that file.
+#
+# After adding your custom requirements you can run one simple command to keep
+# your virtualenv up to date:
+#
+#   pip-sync requirements/local.txt [--dry-run]
```

Comment:
just for my learning, trying to understand different between pip-sync and pip install. Got confused with `-r dev-requirements.txt` in the following line. Please share any link(s) to help understand how this is going to work?

Reply:
`pip install -r requirements.txt` installs new things only. `pip-sync requirements.txt` installs new things _and_ uninstalls things that are no longer required. See also https://github.com/jazzband/pip-tools/#example-usage-for-pip-sync

One could have `local.in` as well and generate `local.txt` from it, but to me that feels like overkill. I have added it to `.gitignore` just in case others see some advantage to that workflow.